website/client/js/controllers/chatCtrl.js

```diff
@@ -114,14 +114,13 @@ habitrpg.controller('ChatCtrl', ['$scope', 'Groups', 'Chat', 'User', '$http', 'A
       });
     };
 
-    $scope.sync = function(group){
-      if(group.type == 'party') {
-        group.$syncParty(); // Syncs the whole party, not just 15 members
-      } else {
-        group.$get();
-      }
-      // When the user clicks fetch recent messages we need to update
-      // that the user has seen the new messages
+    $scope.sync = function(group) {
+      //@TODO: We need to use chat service here
```

Comment:
I think in v2 fetching chat fetched the whole group, let's not change the behavior I think some people used it to update their parties data